app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -1,69 +1,34 @@
-import {
-  Checkbox,
-  FieldSize,
-  IOptionItem,
-  OptionItem,
-  RadioGroup,
-  Select,
-  SelectDate,
-  Text,
-} from 'components/form';
 import { FormPage } from 'components/form/formpage';
 import { IContentModel } from 'hooks/api-editor';
 import React from 'react';
 import { useNavigate } from 'react-router-dom';
 import { SortingRule } from 'react-table';
 import ReactTooltip from 'react-tooltip';
-import { useApp, useContent, useLookup } from 'store/hooks';
-import { initialContentState } from 'store/slices';
-import { Button, ButtonVariant, Loader } from 'tno-core';
-import { Col, Row } from 'tno-core/dist/components/flex';
+import { useApp, useContent } from 'store/hooks';
+import { Button, ButtonVariant } from 'tno-core';
 import { Page, PagedTable } from 'tno-core/dist/components/grid-table';
-import { getSortableOptions, getUserOptions } from 'utils';
 
-import { columns, defaultFilter, defaultPage, fieldTypes, timeFrames } from './constants';
+import { ContentFilter } from '.';
+import { columns, defaultPage } from './constants';
 import { IContentListFilter } from './interfaces';
 import * as styled from './styled';
 import { makeFilter } from './utils';
 
 export const ContentListView: React.FC = () => {
   const [{ userInfo, requests }, { isUserReady }] = useApp();
   const userId = userInfo?.id ?? '';
-  const [{ contentTypes, mediaTypes, users }] = useLookup();
-  const [{ filter, filterAdvanced }, { findContent, storeFilter, storeFilterAdvanced }] =
-    useContent({ filter: { ...defaultFilter, userId } });
+  const [{ filter, filterAdvanced, content }, { findContent, storeFilter }] = useContent();
   const navigate = useNavigate();
 
-  const [mediaTypeOptions, setMediaTypeOptions] = React.useState<IOptionItem[]>([]);
-  const [contentTypeOptions, setContentTypeOptions] = React.useState<IOptionItem[]>([]);
-  const [userOptions, setUserOptions] = React.useState<IOptionItem[]>([]);
-  const [page, setPage] = React.useState(defaultPage);
-  const [timeframe, setTimeframe] = React.useState(timeFrames[Number(filter.timeFrame)]);
-
-  const printContentId = (contentTypeOptions.find((ct) => ct.label === 'Print')?.value ??
-    0) as number;
+  // Set the page for the grid table.
+  const page = !!content
```

Comment:
We now store a page in redux so that we can iterate through it on the content form.